entrypoint/versionbump/versionbump_suite_test.go

```diff
@@ -27,6 +27,7 @@ func TestMain(m *testing.M) {
 	flag.Var(&customflag.TestCaseNameFlag, "testCase", "Comma separated list of test case names to run")
 	flag.StringVar(&customflag.ServiceFlag.TestConfig.WorkloadName, "workloadName", "", "Name of the workload to a standalone deploy")
 	flag.BoolVar(&customflag.ServiceFlag.TestConfig.DeployWorkload, "deployWorkload", false, "Deploy workload customflag for tests passed in")
+	flag.BoolVar(&customflag.ServiceFlag.TestConfig.DeleteWorkload, "deleteWorkload", false, "Delete workload customflag for tests passed in")
```

Comment:
question. do you need a delete workload var on the cli why ? 
if u already have params to delete and apply on the tcs that uses it.

this deploy workload flag its to deploy any extra workload that your tests might not being deploying. 
or any other yaml that u need to complement a test such as bandwidth-annotations.yaml

Reply:
We don't technically need the delete workload var on the cli. At first I had it hard coded to `true` to just delete every workload on these tests, but I figured in some cases people might want to leave the workload around, so I changed it to be the flag. I'm not picky about the implementation though -- we can remove and hard code to either true or false